include/xsimd/arch/xsimd_wasm.hpp

```diff
@@ -1039,6 +1104,44 @@ namespace xsimd
             return wasm_f64x2_extract_lane(tmp2, 0);
         }
 
+        // reduce_max
+        template <class A, class T, class _ = typename std::enable_if<(sizeof(T) <= 2), void>::type>
```

Comment:
Isn't this exactly the same code as the one we have for the generic implementation?